### PR TITLE
fix: fixed application properties linking data service

### PR DESCRIPTION
**Pull request by dgauldie**
the production version of the hmi-server was not configured to connect to the data-service. This is now fixed.

Also, I renamed all the .yml files to .yaml for consistency. I hated that we had both versions in the repo.

# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Resolves #(issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide reproduction instructions and list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by YohannParis — DISMISSED** [review]
LGTM

File: packages/services/hmi-server/src/main/resources/META-INF/resources/index.html [hunk 1]
```diff
@@ -1,285 +1,359 @@
...
...Er73Nem+yiICRXzJ5qWlWl5S+gQMe3J8SUJQ4C/FfOn4ZF6S6x3zJZOX8H4c8SXKS+R7nwq+VOpf2mnzEvtLzX1vtd9beIq5x12i5iXDlz6UwBTlpe/4/Th5kJfcPY77vWve44q8RL73GvK9qU/g05AvfRzxpVflHqfyJfjedI9zfGna5SW5x8V8aSXxJZsS4HvbOoGVhjWV8hJ8b+0eJ7/ORH0C8Jea93tP1PW9v4r8JQlMFf1LSb60nP2l9Zv6sYdCfMn63gP96n7cyep+3FTJ99b8JclL2I/L9Ql8C99b6RNg3zvbJ9B8P+6VRvtx3l9Cv/e8fTXfe0vX974/4ktPZ/bj0CeAvkrje3O/93rqE1B97yxf0vd2LV8KfG+7h/Klvh93ULAfhz6BgC81248DX6J7nO+rXMh8Cb635CaFL8H35n7vXnuPQ59AeT8u0yewZ3yP6wn4kjyWL5mnxn7cbuj3TvAlcexb5C8pfQJx/9KI34+Tp+F+XB3f+/2SvyTPsM1LVx5u8xL8pYPBl/R+70H5B7p5ye/HgS85fynaj/P+kstL6Pc+BXxJy0sjvB83VOJLS9W8BH/pPt33vsX4S9X3uFsK33vS+d7H4x4X8SV/j4O/9CDyEvhS0l9CXlp7TgO+lO8T+GYW/hL73j4vLa7OSxKYHF5K86XvwJfIX1L4Et/jmC/tMnyJ/KU1ob+0JvKXxPfW/CXkpT8dXzo/9L1vJ99b5UvwvdW8xL43+JJ8nOwTELpk+RLf44Qv6f4S+gR03/t07MdNoK8ys4dSzku0H0e+9zF6n8De3l9ab/oE9H7vDvUJrARfsn0C2f24Ked76/tx7HujrxJ8ifbjcv4S86Wc7019Ao3345bYvPS7jUvymD4B+UUTfQLBftz8nO/9cOIeh/04k5c2oN+75C/12/04ITS0H0e+d+0+AbGXmuzHgS+R753fj2uZPoElni9hP+7Yiv2469An0EG/d0/Qv0S+d0/VflyH+yp7oj6Bnl7aj2O+NPf9OOQl7hPoqH0CyEtpviRfpnyL8j22+B6X2o8b/Ef7cYNz2I9L5iW5x4nvLbxL7V+K+NIfb+f50lGnoE9gyPEledK+t2RFxfcmvpT2l0ZPqn+Pkzyj+t7H2z4BLS/JU+kvfabf4x6ke1w1XzqV8hL4kuovCWCaQV5q7HsX6cDwpaeq7nGSl2YL
```
Comment (dvince2) [inline packages/services/hmi-server/src/main/resources/META-INF/resources/index.html:318]: Is it part of the Quarkus license to include this boiler-plate text?  Or can/should it be removed?

**Review by dvince2 — APPROVED** [review]
Approving - minor question about boiler-plate quarkus in `index.html` but not a blocker.

File: packages/services/hmi-server/src/main/resources/META-INF/resources/index.html (hunk 1 above)
Comment (dgauldie) [reply packages/services/hmi-server/src/main/resources/META-INF/resources/index.html:318]: this is the Quarkus dev page. When you run quarkus in dev mode, you can change configurations on the fly using this interface. It's not in production

**Review by dvince2 — APPROVED** [review]

**Pull request merged**